src/planner/parabolic/ParabolicUtil.cpp

```diff
@@ -120,6 +120,27 @@ bool checkStateSpace(const statespace::StateSpace* _stateSpace)
   }
 }
 
+Eigen::VectorXd getTangentVector(
```

Comment:
It seems that there is some overlap between this function and `GeodesicInterpolator::getTangentVector`. Maybe we should put both functions somewhere in `aikido::statespace`?